modules/core/src/test/scala/org/tessellation/domain/cell/L0CellSuite.scala

```diff
@@ -0,0 +1,93 @@
+package org.tessellation.domain.cell
+
+import cats.data.{NonEmptyList, NonEmptySet}
+import cats.effect.IO
+import cats.effect.std.Queue
+
+import scala.collection.immutable.SortedSet
+
+import org.tessellation.dag.domain.block.{BlockReference, DAGBlock, L1Output}
+import org.tessellation.kernel.{Cell, CellError}
+import org.tessellation.schema.ID.Id
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.height.Height
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.transaction._
+import org.tessellation.sdk.domain.collateral.Collateral
+import org.tessellation.security.hash.{Hash, ProofsHash}
+import org.tessellation.security.hex.Hex
+import org.tessellation.security.signature.Signed
+import org.tessellation.security.signature.signature.{Signature, SignatureProof}
+
+import eu.timepit.refined.auto._
+import weaver.SimpleMutableIOSuite
+import weaver.scalacheck.Checkers
+
+object L0CellSuite extends SimpleMutableIOSuite with Checkers {
+
+  private val peerId1 = PeerId(
+    Hex(
+      "6128e64d623ce4320c9523dc6d64d7d93647e40fb44c77d70bcb34dc4042e63cde16320f336c9c0011315aa9f006ad2941b9a92102a055e1bcc5a66ef8b612ef"
+    )
+  )
+
+  private val peerId2 = PeerId(
+    Hex(
+      "79c4a78387a8782dbc88de95098d134a7dbf3b8a3316eaa1e41e112dc5b21a5b0cefdd0871495435591089264aa5c8a2429a75b384519662184bedfa6e7b886f"
+    )
+  )
+
+  def generateL1Output() = {
+    val signedTxn = Signed(
+      Transaction(
+        Address("DAG0y4eLqhhXUafeE3mgBstezPTnr8L3tZjAtMWB"),
+        Address("DAG07tqNLYW8jHU9emXcRTT3CfgCUoumwcLghopd"),
+        TransactionAmount(1L),
+        TransactionFee(1L),
+        TransactionReference(TransactionOrdinal(1L), Hash("hash")),
+        TransactionSalt(0L)
+      ),
+      NonEmptySet.one(SignatureProof(Id(Hex("")), Signature(Hex(""))))
+    )
```

Comment:
I think you can use a generator for those data

Reply:
Generating Addresses may be hard.